.github/workflows/macos.yml

```diff
@@ -19,17 +19,21 @@ on:
         type: string
       # System configuration
       osversion_xcodeversion_toxenv_tuples:
+        # As of 2024-02, "runs-on: macos-latest" is macos-12.
+        # and "runs-on: macos-14" selects the new M1 runners.
+        # https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories
         description: 'Stringified JSON object'
         default: >-
-          [["latest", "",             "homebrew-macos-usrlocal-minimal"],
-           ["latest", "",             "homebrew-macos-usrlocal-standard"],
-           ["11",     "xcode_13.2.1", "homebrew-macos-usrlocal-minimal"],
+          [["11",     "xcode_13.2.1", "homebrew-macos-usrlocal-minimal"],
+           ["12",     "",             "homebrew-macos-usrlocal-minimal"],
            ["12",     "",             "homebrew-macos-usrlocal-standard"],
+           ["12",     "",             "homebrew-macos-usrlocal-python3_xcode-standard"],
+           ["12",     "",             "homebrew-macos-usrlocal-maximal"],
            ["13",     "xcode_15.0",   "homebrew-macos-usrlocal-standard"],
-           ["latest", "",             "homebrew-macos-usrlocal-maximal"],
-           ["latest", "",             "homebrew-macos-usrlocal-python3_xcode-standard"],
+           ["14",     "",             "homebrew-macos-opthomebrew-standard"],
            ["latest", "",             "conda-forge-macos-minimal"],
```

Comment:
This fails, can we disable it until someone finds the time to fix it?

Reply:
No, it's useful as a reminder what needs fixing.

Reply:
An issue would serve that purpose better.

Reply:
Nope. Source: I'm the person who monitors the portability CI.